### PR TITLE
adding a simple parallel sleep and fix validation with pytorch

### PR DESCRIPTION
**Pull request by kannon92**
I was trying to run the resnet example and I notice that we were missing the network field.  

This may not be necessary with other work in place but figured I'd open something up.

I also added a lightweight example just for testing purposes.

### CODE REVIEW

File: examples/pytorch/resnet.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ metadata:
 spec:
   replicatedJobs:
   - name: workers
+    network:
```
[the remark this answers is not in the document]
Comment (kannon92) [reply examples/pytorch/resnet.yaml:8]: Yea, it says that network is required when I tried to apply the example.

[the remark this answers is not in the document]
Comment (ahg-g) [reply examples/pytorch/resnet.yaml:8]: strange, I ma not getting that error. Can you send a PR to add optional and omitempty on this and the exclusive fields? also the namespaceSelector inside exclusive

[the remark this answers is not in the document]
Comment (kannon92) [reply examples/pytorch/resnet.yaml:8]:
```
$ kubectl apply -f examples/pytorch/resnet.yaml 
error: error validating "examples/pytorch/resnet.yaml": error validating data: ValidationError(JobSet.spec.replicatedJobs[0]): missing required field "network" in io.x-k8s.jobset.v1alpha1.JobSet.spec.replicatedJobs; if you choose to ignore these errors, turn validation off with --validate=false
```

I'll open a PR for those.

[the remark this answers is not in the document]
Comment (kannon92) [reply examples/pytorch/resnet.yaml]: Is there any point in having job templates be optional?

[the remark this answers is not in the document]
Comment (ahg-g) [reply examples/pytorch/resnet.yaml]: > Is there any point in having job templates be optional?

I don't think so, no.

**Pull request merged**